Answers.md

```diff
@@ -1 +1,8 @@
-
+1. .map() allocates memory and stores return values while .foreach() returns undefined and throws away return values.
```

Comment:
Main point here is that map creates a new array and forEach iterates over existing.